### ISSUE TITLE
websocket controlling loop crash

### ISSUE THREAD

**Issue by linnik** [sockjs/sockjs-erlang#14]
sockjs-erlang with misultin produces such error during http://localhost:8080/example-cursors.html websockets test.

other transports are ok.

```
=ERROR REPORT==== 30-Dec-2011::12:39:41 ===
Error in process <0.84.0> with exit value: {badarg,[{jiffy,nif_decode,["\"{\\"x\\":188,\\"y\\":22,\\"t\\":1325266781650,\\"id\\":\\"0649712570358062\\"}\""],[]},{jiffy,decode,1,[{file,"src/jiffy.erl"},{line,11}]},{sockjs_util,jiffy,2,[{file,"src/sockjs_util.erl"},{line,39}]},{sockjs_http... 


=ERROR REPORT==== 30-Dec-2011::12:39:41 ===
        module: misultin_websocket
        line: 238
linked websocket controlling loop crashed with reason: {badarg,
                                                        [{jiffy,nif_decode,
                                                          ["\"{\\\"x\\\":188,\\\"y\\\":22,\\\"t\\\":1325266781650,\\\"id\\\":\\\"0649712570358062\\\"}\""],
                                                          []},
                                                         {jiffy,decode,1,
                                                          [{file,
                                                            "src/jiffy.erl"},
                                                           {line,11}]},
                                                         {sockjs_util,jiffy,
                                                          2,
                                                          [{file,
                                                            "src/sockjs_util.erl"},
                                                           {line,39}]},
                                                         {sockjs_http,
                                                          misultin_ws_loop0,3,
                                                          [{file,
                                                            "src/sockjs_http.erl"},
                                                           {line,129}]}]}
```

**Comment by squaremo** [comment]
Is this on sockjs/master?

**Comment by squaremo** [comment]
Yes it is, I see it. I've fixed it elsewhere (in my fork) but I will fix it here too.

**Comment by squaremo** [comment]
Fixed in commit fb32ebb
